controllers/dbconfig.php

```diff
@@ -4,6 +4,10 @@ class DbconfigController extends ControllerBase {
         protected function dbInit() {}
 
         public function create( $user, $pass, $dbname ) {
+            if ( $this->environment == 'production' ) {
+                go(); // disable in production
+            }
+            

```

Comment:
This can be abstracted into `ControllerBase` as `$this->disableInProduction()` or `$this->developmentOnly()`.